lib/address_space/add-event-type/install.js

```diff
@@ -5,57 +5,58 @@
 
 import assert from "better-assert";
 import _ from "underscore";
+import { randomBytes } from "crypto";
```

Comment:
Was there a reason these were imported inside of the function? I pulled them out.